packages/qdrant-loader/src/qdrant_loader/connectors/publicdocs/connector.py

```
@@ -176,11 +177,33 @@ def _should_process_url(self, url: str) -> bool:
+            source_type=self.config.source_type,
+            source=self.config.source,
+            url=page,
+            # For public docs, we don't have a created or updated date. So we use a very old date.
+            # The content hash will be the same for the same page, so it will be update if the hash changes.
+            created_at=datetime(1970, 1, 1, 0, 0, 0, 0, UTC),
+            updated_at=datetime(1970, 1, 1, 0, 0, 0, 0, UTC),
+        )
 
-        Returns:
-            List of documents
+    async def stream_documents(
```

Ruff (0.15.15)

[warning] Too many branches (13 > 12)

(PLR0912)